schema/va-spec/base/pathogenicity-evidenceline-profile-source.yaml

```diff
@@ -0,0 +1,129 @@
+          may be pathogenic for a particular disease).      
+      specifiedBy:
+        extends: specifiedBy
+        description: >-
+          A method that specifies how evidence items used in the Evidence Line are to be evaluated
+          and weighed as evidence for or against the pathogenicity of the assessed variant.
+    # required:        # TO DO: decide which if any attributes should be *required* in this profile. 
+
+  # Variant Pathogenicity Proposition
+  VariantPathogenicityProposition:
+    inherits: gks-core:Proposition
+    maturity: draft
+    type: object
+    description: >-
+      A proposition describing the role of a variant in causing an inherited condition.
+    properties:
+      type:
+        extends: type
+        const: VariantPathogenicityProposition
+        default: VariantPathogenicityProposition
+        description: Must be "VariantPathogenicityProposition"
+      subjectVariant:
+        extends: subject 
+        oneOf:
+          - $ref: "/ga4gh/schema/vrs/2.x/json/Variation"
+          - $ref: "/ga4gh/schema/cat-vrs/1.x/json/CategoricalVariant"
+          - $ref: "/ga4gh/schema/gks-core/1.x/json/iriReference"
+        description: A variant that is the subject of the Statement.
+      predicate:
+        extends: predicate
+        const: isCausalFor
+      objectCondition:
+        extends: object
+        oneOf:
+          - $ref: "/ga4gh/schema/va-spec/1.x/base/json/Condition"
+          - $ref: "/ga4gh/schema/gks-core/1.x/json/iriReference"
+        description: The :ref:`Condition` for which the variant impact is stated.
+      penetranceQualifier:
+        type: string
+        enum:
+          - high
+          - low
+          - risk allele
+        description: >-
+          Reports the penetrance of the pathogenic effect - i.e. the extent to which the
+          variant impact is expressed by individuals carrying it as a measure of the
+          proportion of carriers exhibiting the condition. 
+      modeOfInheritanceQualifier:
+        type: array
+        ordered: false
+        items:
+          $ref: "/ga4gh/schema/gks-core/1.x/json/Coding"
+        description: >-
+          Reports a pattern of inheritance expected for the pathogenic effect of the variant.
+          Use HPO terms within the hierarchy of 'HP:0000005' (mode of inheritance) to specify.
+      geneContextQualifier:
+        description: >-
+          Reports the gene through which the pathogenic effect asserted for the variant is mediated
+          (i.e. it is the variant's impact on this gene that is responsible for causing the condition).
+        oneOf:
+          - $ref: "/ga4gh/schema/gks-core/1.x/json/MappableConcept"
+          - $ref: "/ga4gh/schema/gks-core/1.x/json/iriReference"  
+    required:
+      - subjectVariant
+      - predicate
+      - objectCondition
+
+     # TO DO: decide if we want to include the qualifiers defined on the VarPathStatement class in the VarPathProposition
```

Comment:
@mbrush I went for it and added in the `qualifiers` from VarPathStatement so we could see it first-hand. It starts making things pretty clear that we "probably" will move to using 'targetProposition' on `Statement` and forego the flattening of these fields directly into Statement. As it stands we will need to always make sure this schema stays in lock step with it's sibling Statement SPOQ attributes specs. (TRICKY!) ... @ahwagner should we?

Reply:
Holy crap. If I am understanding you right, @larrybabb, you just sold me on this. I think that you are suggesting that we move ALL of the SPOQ attributes to propositions, making these propositions reusable across statement types and evidence lines, with the consequence that BOTH evidence lines and statement classes become concrete entities that don't need subclassing (because all of the domain-specific properties live in the typed proposition).

Am I understanding that right!? If so, this is a solid design improvement and I am in favor.